spec/system/task_spec.rb

```diff
@@ -111,4 +113,11 @@ def visit_with_http_auth(path)
     password = 'guest_password'
     visit "http://#{username}:#{password}@#{Capybara.current_session.server.host}:#{Capybara.current_session.server.port}#{path}"
   end
+
+  def user_login
+    visit new_session_path
+    fill_in 'session-email', with: 'user@example.com'
```

Comment:
メールアドレスは固定値を設定するより、引数で受け取ったemailでログインするようにできれば、adminユーザーでのログインにも利用できるようになります。
```suggestion
  def login_as(user)
    visit new_session_path
    fill_in 'session-email', with: user.email
```

Reply:
9f0c135 で修正確認